src/unit_tests/monitord/test_monitor_actions.c

```diff
@@ -0,0 +1,384 @@
+/*
+ * Copyright (C) 2015-2020, Wazuh Inc.
+ * November, 2020.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <cmocka.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "../wrappers/common.h"
+#include "../wrappers/wazuh/shared/debug_op_wrappers.h"
+#include "../wrappers/wazuh/shared/mq_op_wrappers.h"
+#include "../wrappers/wazuh/shared/hash_op_wrappers.h"
+#include "../wrappers/wazuh/os_net/os_net_wrappers.h"
+
+#include "headers/store_op.h"
+#include "monitord/monitord.h"
+#include "headers/defs.h"
+#include "headers/shared.h"
+#include "config/config.h"
+
+time_t __wrap_time(__attribute__((unused)) time_t *t) {
+    return mock_type(time_t);
+}
+
+char* __wrap_get_agent_id_from_name(__attribute__((unused)) char *agent_name) {
+    return mock_type(char*);
+}
+
+int __wrap_auth_remove_agent(__attribute__((unused)) int sock, const char *id, __attribute__((unused)) int json_format) {
+    check_expected(id);
+    return mock();
+}
+
+int __wrap_auth_connect() {
```

Comment:
Shouldn't we create these wrappers inside an independent module?

Reply:
It was created!